requirements.txt

```diff
@@ -8,3 +8,5 @@ Babel>=1.3
 tortilla>=0.4.1
 Jinja2>=2.7.3
 Sphinx>=1.2.3
+docopt>=0.6.2
+simplejson>=3.8.1
```

Comment:
Each time you change the requirements, we should create a ticket to change the deps in the package to build.